doc/apiv3-documentation.apib

```diff
@@ -114,12 +115,17 @@ For example a work package without an assignee will still have an assignee link,
 
 If a `title` is present, the client can display the title to the user when referring to the resource.
 
-Templated links are links that contain client replacable parts. Replaceable parts are enclosed in braces. For example
+Templated links are links that contain client replaceable parts. Replaceable parts are enclosed in braces. For example
 the link `api/v3/example/{id}` is not complete in itself, but the client needs to replace the string `{id}` itself.
 As of now the API does not indicate the valid replacement values.
 
 The `method` indicates which HTTP verb the client *must* use when following the link for the intended purpose.
 
+If a `payload` is specified, it needs to be sent as the body of the request to achieve the desired result (e.g. perform the
+action represented by the link). If no `payload` is specified, there is either no payload to send or there are multiple valid
+payloads. A payload might also be templated slightly. If the `templated` property is true, a payload might contain replaceable parts
```

Comment:
It is not fully templatable, I only added semantics for templating inside strings in the payload.

If you know a better word, tell me...